runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/HotColdClue.java

```diff
@@ -78,6 +79,12 @@
 	@Nullable
 	private HotColdSolver hotColdSolver;
 	private WorldPoint location;
+	/**
+	 * This is only used for dev mode through the ::hotcold command.
+	 * This is the location that we are looking for.
+	 * Default is by the Baxtorian Falls so we can load the interface without having to set the location on load.
+	 */
+	private HotColdLocation devModeLocation = HotColdLocation.KANDARIN_BAXTORIAN_FALLS;
```

Comment:
This can probably just be global, you cannot hotcold solve more than one clue at once